test/plugins/foreign/schema-bare-foreign.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx h */
+
+export default editor => editor.normalize()
+
+export const input = <value>
+    <document>
+        <foreign>Foreign</foreign>
+    </document>
+</value>
+
+export const output = <value>
+    <document>
+        <p><text/></p>
+    </document>
+</value>
```

Comment:
This term should be moved into the paragraph, not removed.

Reply:
There is only `normalizeDocument child_min_invalid` returned as an error for this test.
It might be because `document` can contain only `block` elements. 
What would you like me to do with this test?